backend/applications/middleware.go

```diff
@@ -33,6 +34,14 @@ func ApplicationMiddleware(
 	}
 }
 
+// Support app version with old application name kids
```

Comment:
In general it's better to only comment why you're doing something and not the what.
For the What it's better to refactor the code and make it easier to understand.